packages/dds/tree/src/test/simple-tree/proxies.spec.ts

```diff
@@ -6,9 +6,12 @@
 import { strict as assert } from "assert";
 import { MockHandle } from "@fluidframework/test-runtime-utils";
 import { TreeArrayNode, NodeFromSchema, SchemaFactory } from "../../simple-tree/index.js";
+// TODO: test other things from "proxies" file.
```

Comment:
Do we have an item filed for this?

Reply:
Most of our 500+ TODOs in this package don't have items. I think ones like this that are tracking a minor code quality issue and not a bug arn't worth littering our public codebase with internal work item references, and can't be tracked on github as stalebot would close them before we fixed it in most cases.